.ci/bwcVersions

```diff
@@ -83,4 +83,5 @@ BWC_VERSION:
   - "2.11.2"
   - "2.12.0"
   - "2.12.1"
+  - "2.13.0"
```

Comment:
@reta Should this also have "2.13.1"?

Reply:
@kotwanikunal very right, added, thank you!

Reply:
Thanks :)